src/Components/Locations.js

```diff
@@ -0,0 +1,59 @@
+import axios from "axios";
+import React, { Component } from "react";
+import "./Locations.css";
+
+export class Locations extends Component {
+  constructor() {
+    super();
+    this.state = {
+      showLocations: false,
+      failedApiCall: false,
+      locations: [],
+    };
+  }
+
+  componentDidMount = () => {
+    this.getLocations();
+  };
+
+  getLocations = async () => {
+    try {
+      const { data } = await axios.get(
+        `https://ghibliapi.herokuapp.com/locations`
+      );
+      this.setState({
+        locations: data.map((location) => location.name),
```

Comment:
Nice mapping! Makes your mapping below much cleaner since it's only dealing with names now. Well done!